packages/arb-token-bridge-ui/tests/e2e/specs/approveToken.cy.ts

```diff
@@ -7,7 +7,7 @@ import {
   getL2NetworkName
 } from '../../support/common'
 
-const ERC20TokenAddressL1 = Cypress.env('ERC20_TOKEN_ADDRESS_L1')
+const ERC20TokenAddressL1 = Cypress.env('ERC20_TOKEN_ADDRESS_PARENTCHAIN')
 
 describe('Approve token and deposit afterwards', () => {
```

Comment:
i just realised that this test was never correctly described
it should be 
```suggestion
describe('Approve token for deposit', () => {
```

Reply:
Updated